iceberg/iceberg-catalog/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -364,7 +413,6 @@ private void setHmsTableParameters(String newMetadataLocation, Table tbl, TableM
     tbl.setParameters(parameters);
   }
 
-  @VisibleForTesting
```

Comment:
Why is this annotation removed? I see the same in upstream Iceberg code.

Reply:
It was a mistake taken at porting the Hive: Refactor commit lock mechanism from HiveTableOperations change. I fixed it. 

Thank you for finding this.